build_ubuntu.sh

```diff
@@ -29,7 +31,7 @@ source $HOME/.cargo/env
 
 # Install snarkOS
 # cargo clean
-cargo install --locked --path .
+cargo install --locked --path . --features cuda
```

Comment:
This should not be the default for all users. Feel free to update the README for provers